ui/src/app/pages/workspace/workspace-about.spec.tsx

```diff
@@ -325,4 +326,54 @@ describe('WorkspaceAbout', () => {
       workspace.namespace
     );
   });
+
+  it('renders Community Workspace section when enabled', () => {
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enablePublishedWorkspacesViaDb: true },
+    });
+    component();
+    expect(screen.getByText('Community Workspace')).toBeInTheDocument();
+  });
+
+  it('does not render Community Workspace section when enablePublishedWorkspacesViaDb disabled', () => {
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enablePublishedWorkspacesViaDb: false },
+    });
+    component();
+    expect(screen.queryByText('Community Workspace')).not.toBeInTheDocument();
+  });
+
+  it('Publish button is enabled for workspace owner', () => {
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enablePublishedWorkspacesViaDb: true },
+    });
+    component();
+    const publishButton = screen.getByText('Publish');
+    expect(publishButton).toBeEnabled();
```

Comment:
Our button elements do not make use of the disabled attribute but for testing, we rely on what their cursor style is.

We have two helper functions for this:
expectButtonElementDisabled
expectButtonElementEnabled